src/main/webapp/dev/js/common/bootboxWrapper.es6

```diff
@@ -32,8 +32,12 @@ function showModalAlert(titleText, messageText, okButtonText, color) {
             },
         },
     })
-    // applies bootstrap color to title background
-    .find('.modal-header').addClass(`alert-${color}` || StatusType.DEFAULT);
+    .find('.modal-header')
+        // applies bootstrap color to title background
+        .addClass(`alert-${color}` || StatusType.DEFAULT)
```

Comment:
I was actually going to ask you about this but then I forgot! But if this really was the case, then the current CommonJsTest in the master should have failed as it explicitly expected an `alert-info` class when calling `setStatusMessage` without setting the color property (ie leaving it undefined). See [this line.](https://github.com/TEAMMATES/teammates/blob/master/src/main/webapp/dev/js/test/CommonJsTest.es6#L211)